nodejs/week2/warmup/index.js

```diff
@@ -0,0 +1,54 @@
+const express = require('express');
+const app = express();
+const port = 3001;
+
+app.get('/', (request, response) => {
+    response.send("Example app listening on port 3001!")
+});
+
+/*
+GET /numbers/add?first=<number here>&second=<number here>. In response send sum (first + second).
+GET /numbers/multiply/<first number here>/<second number here>. in response send multiplication (first * second).
+Example: going to localhost:3000/numbers/add?first=3&second=7 should respond with 10
+Hints:
+Find express documentation online.
+Use req.query and req.params properties on req object.
+*/
+
+//USing req.Query
+app.get('/numbers/add', (request, response) => {
+    const theSum = Number(firstNumber) + Number(secondNumber);
+    if (firstNumber && secondNumber) {
+        response.send(`Input a number`);
+    } else {
+        response.send(`the answer is ${theSum}`);
+    }
+});
+
+app.get('/numbers/multiply', (request, response) => {
+    const multiplication = Number(firstNumber) * Number(secondNumber);
+    if (firstNumber && secondNumber) {
```

Comment:
Here you could rather write that if request.query.firstNumber or request.query.secondNumber is undefined, send this response.
Right now you are telling that if both of the numbers exist (they are in the url), then send the response 'Input a number'.